tests/garage/envs/bullet/test_bullet_env.py

```diff
@@ -88,3 +79,5 @@ def test_pickle_creates_new_server(env_ids):
                 # Some environments have _p as the pybullet module, and they
                 # don't store client id, so can't check here
                 pass
+        for env in envs:
```

Comment:
i think pre-commit with catch this, but you should always leave a trailing line after a block